### PR TITLE
Add footer link to manual with configurable URL

### PR DESCRIPTION
**Pull request by relaxnow**
https://www.pivotaltracker.com/projects/1163646/stories/114385851

@rjkip to start your monday morning with 😄

### CODE REVIEW

File: app/Resources/translations/messages.en_GB.xliff [hunk 1]
```diff
@@ -1044,9 +1049,9 @@
         <target>Token type disabled</target>
       </trans-unit>
       <trans-unit id="8e4a356a46fe95b04ae1cbb201b8d62e15670b2e" resname="ra.vetting.sms.challenge_body">
-        <jms:reference-file line="199">RaBundle/Service/VettingService.php</jms:reference-file>
+        <jms:reference-file line="209">RaBundle/Service/VettingService.php</jms:reference-file>
         <source>ra.vetting.sms.challenge_body</source>
-        <target>Your code: %challenge%</target>
+        <target>ra.vetting.sms.challenge_body</target>
```
Comment (rjkip) [inline app/Resources/translations/messages.en_GB.xliff]: :goberserk:

File: app/Resources/translations/messages.nl_NL.xliff [hunk 2]
```diff
@@ -1044,9 +1049,9 @@
         <target>Tokentype uitgeschakeld</target>
       </trans-unit>
       <trans-unit id="8e4a356a46fe95b04ae1cbb201b8d62e15670b2e" resname="ra.vetting.sms.challenge_body">
-        <jms:reference-file line="199">RaBundle/Service/VettingService.php</jms:reference-file>
+        <jms:reference-file line="209">RaBundle/Service/VettingService.php</jms:reference-file>
         <source>ra.vetting.sms.challenge_body</source>
-        <target>Uw SMS-code: %challenge%</target>
+        <target>ra.vetting.sms.challenge_body</target>
```
Comment (rjkip) [inline app/Resources/translations/messages.nl_NL.xliff]: :japanese_goblin:

File: app/Resources/views/base.html.twig [hunk 3]
```diff
@@ -88,6 +88,9 @@
 
         {% block footer %}
             <hr>
+            <ul id="footerLinks">
```
Comment (rjkip) [inline app/Resources/views/base.html.twig]: Please use kebab case for this ID as well.

**Comment by rjkip** [comment]
And a free rhyme for you:

> Easy for thee, hard for me
> Please include a screenshot
> So I can those footer links see

**Comment by rjkip** [comment]
Looked at the issue, I should have.

df

**Comment by rjkip** [comment]
Please verify with SURF the styling is indeed what they want (esp. since they requested left corner :stuck_out_tongue_winking_eye:)

File: app/Resources/views/base.html.twig (hunk 3 above)
Comment (relaxnow) [inline app/Resources/views/base.html.twig]: Looking at https://github.com/SURFnet/Stepup-RA/blob/6cd272c55f0e3ec962324cd66c6529a373720a4d/src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig#L61-L61 I figured we were doing camelCase for ids, snake for classes?

**Comment by relaxnow** [comment]
Damnit I totally read over that, fixed, thanks!

File: app/Resources/translations/messages.nl_NL.xliff (hunk 2 above)
Comment (relaxnow) [inline app/Resources/translations/messages.nl_NL.xliff]: Bloody _trans. Fixed, thanks.

File: app/Resources/translations/messages.en_GB.xliff (hunk 1 above)
Comment (relaxnow) [inline app/Resources/translations/messages.en_GB.xliff]: Bloody _trans. Fixed, thanks.

File: app/Resources/views/base.html.twig (hunk 3 above)
Comment (rjkip) [inline app/Resources/views/base.html.twig]: I think that is the result of a lapse of PR judgement on my part. Let's do kebab case for both.

**Comment by relaxnow** [comment]
@rjkip could you take another gander?

**Comment by rjkip** [comment]
@relaxnow Er staat nog wat commentaar open van SURF :sweat_smile:

**Comment by relaxnow** [comment]
@rjkip ouch, I didn't think to look at how SS does this. Seems it's very strange to have 2 pill menus.
Anyway, I'm (slightly) more confident this solution will be the one.
Third time's the charm, right?

**Comment by rjkip** [comment]
I can't really see the result, but if you're confident it looks good (or confer with our esteemed colleague @arothuis) then merge it. However, I just realised: the URL points to a Dutch page. This is fine by SURF?

**Comment by relaxnow** [comment]
@rjkip it's in the issue, I'll paste it here too:
screen shot 2016-06-14 at 13 18 30

Damnit, you make a good point about the URL! I'll confer with SURF.

**Comment by relaxnow** [comment]
@pmeulen said 'A configurable URL is fine, we can always make a dual language page on the wiki or wherever'

**Pull request merged**